lib/server.js

```diff
@@ -208,6 +209,51 @@ function listenPromise (server, listenOptions) {
   })
 }
 
+/**
+ * Creates a function, based upon initial configuration, that will
+ * be used to verify every incoming request conforms to allowed
+ * HTTP versions for the Fastify instance, e.g. a Fastify HTTP/1.1
+ * server will not serve HTTP/2 requests upon the result of the
+ * verification function.
+ *
+ * @param {object} options
+ * @param {function} [options.serverFactory] If present, the
+ * validator function will skip all checks.
+ * @param {boolean} [options.http2 = false] <need a description>
```

Comment:
Please add the appropriate description. It was unclear to me what the description should be.